packages/cli/src/command.js

```diff
@@ -29,6 +28,11 @@ export default class NuxtCommand {
   }
 
   run() {
+    if (this.argv.help || this.argv.h) {
+      this.showHelp()
+    } else if (this.argv.version || this.argv.v) {
+      this.showVersion()
+    }
     return this.cmd.run(this)
```

Comment:
@galvez @pi0 I noticed in the preliminary documentation that the example there also used a `process.exit(0)`. But the node [docs](https://nodejs.org/dist/latest-v11.x/docs/api/process.html#process_process_exit_code) advise against using process.exit() if possible. Is it an idea to use the [unreffed timeout trick](https://github.com/nuxt/nuxt.js/pull/4239/files#diff-f24635a18a996e327cd2c5124f2e440dR104) to detect whether the run command exited nicely? That way we adhere more to the official Node.js docs/advise instead of promoting the use of forced exits even in non-error conditions.

Reply:
I like that @pimlie -- worth looking into @pi0 :)

Reply:
@pimlie Yes, I agree. Honestly, I was surprised too when saw `process.exit` codes inside `NuxtCommand` 😆 We can limit process exiting logic to `cli/run` command only or use better techniques there like the one you mentioned.

Reply:
Sorry @pi0 -- I had no idea :) Thanks for clearing it up.

Reply:
Now, we only have process.exit in `.catch` case of cli binary :)